### PR TITLE
Add help screen

### PR DESCRIPTION
**Pull request by vinaysshenoy**
Pivotal Story Link: https://www.pivotaltracker.com/story/show/164758290

### CODE REVIEW

**Comment by vinaysshenoy** [comment]
@rakshakhegde This is a large PR. I'm sorry about this.

This is mostly a clone of the Reports feature which you worked on, so it should hopefully be familiar enough that the bulk of it shouldn't be a problem to read.

File: app/src/main/res/values/colors_v2.xml [hunk 1]
```diff
@@ -32,4 +32,5 @@
 
   <color name="launcher_background">@color/white100</color>
   <color name="facility_search_query_highlight">@color/blue2</color>
+  <color name="status_bar">#0a2e52</color>
```
Comment (rakshakhegde) [inline app/src/main/res/values/colors_v2.xml]: Is this status bar's color not a standard color used across other screens?

File: app/src/main/java/org/simple/clinic/home/HomeScreen.kt [hunk 2]
```diff
@@ -89,4 +93,20 @@ class HomeScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context
...
+
+  fun showHelpButton() {
+    showHelpMenuItem
+        .apply {
+          isEnabled = true
+          isVisible = true
+        }
+  }
+
+  fun hideHelpButton() {
```
Comment (rakshakhegde) [inline app/src/main/java/org/simple/clinic/home/HomeScreen.kt]: Can we combine these two show/hide functions by passing in a boolean, just the way we've done at `HelpScreen#showWebview`?

File: app/src/main/res/values/colors_v2.xml (hunk 1 above)
Comment (vinaysshenoy) [reply app/src/main/res/values/colors_v2.xml]: Nope, there's a separate color in Zeplin only for this: https://zpl.io/aR1dWmp

File: app/src/main/java/org/simple/clinic/home/HomeScreen.kt (hunk 2 above)
Comment (vinaysshenoy) [reply app/src/main/java/org/simple/clinic/home/HomeScreen.kt]: Sure.

**Pull request merged**